Assets/MixedRealityToolkit.Tests/EditModeTests/Services/TestSpatialAwarenessDataProvider.cs

```diff
@@ -0,0 +1,61 @@
+﻿
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Tests.Services
+{
+    internal interface ITestSpatialAwarenessDataProvider : IMixedRealityDataProvider, ITestService, IMixedRealitySpatialAwarenessObserver, IMixedRealitySpatialAwarenessMeshObserver
```

Comment:
>ITestSpatialAwarenessDataProvider [](start = 23, length = 33)

Not really a question for this because I think this is consistent with the rest of the test code, but... any reason why this isn't just like, defined down below (i.e. public class TestSpatialAwarenessDataProvider : TestBaseDataProvider,, IMixedRealityDataProvider, ITestService, IMixedRealitySpatialAwarenessObserver, IMixedRealitySpatialAwarenessMeshObserver?

Reply:
Yea I was just following the existing paradigm. The main benefit though is we can test getting this interface as well as the base interfaces in all the registrar/registry stuff